docs/ops/sequence/AUGRUSequence_10.md

```diff
@@ -0,0 +1,145 @@
+# AUGRUSequence  {#openvino_docs_ops_sequence_AUGRUSequence_10}
+
+**Versioned name**: *AUGRUSequence-10*
+
+**Category**: *Sequence processing*
+
+**Short description**: *AUGRUSequence* operation represents a series of AUGRU cells (GRU with attentional update gate).
+
+**Detailed description**: The difference between *AUGRUSequence* and [GRUSequence](./GRUSequence_5.md) is the additional attention score input `A`, which is a multiplier for the update gate.
+The formula was described in the [paper arXiv:1809.03672](https://arxiv.org/abs/1809.03672).
```

Comment:
@mitruska, it would be really nice if you add calculation scheme for this operation. There can be several versions of AUGRU so we can avoid confusion in the implementation and make others to refer our docs (OpenVINO) instead of we do.

Reply:
Thank you for your review @rkazants. 
Sure, I'm going to add the formula, but with @ceciliapeng2011 we are trying to clarify the order of subtraction/multiplication. In general the [GRUCell_3.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sequence/GRUCell_3.md) formula could be reused, just with additional line describing multiplication of update gate (zt) and attention_score.